tasks/mesh_conf.yml

```diff
@@ -0,0 +1,63 @@
+---
+# yamllint disable rule:line-length
+- name: Set current IP fact for each host
+  set_fact:
+    current_ip: "{{ ansible_default_ipv4.address | d(ansible_default_ipv6.address) }}"
+
+- name: Set IP with prefix register
+  shell: "ip addr show | grep {{ current_ip }} | awk '{print $2}'"
+  register:
+    ip_with_prefix_register
+  changed_when: false
+
+- name: Set net CIDR fact
+  set_fact:
+    current_subnet: "{{ ip_with_prefix_register.stdout | ipaddr('subnet') }}"
+
+- name: Set policies fact
+  set_fact:
+    policies: "{{ conn.policies | rejectattr('cidr', 'equalto', 'default') | list }}"
```

Comment:
Don't use `equalto` here - it isn't supported by ansible 2.8/jinja 2.7 - instead use `match`:
`rejectattr('cidr', 'match', '^default$')`